spec/app/ics-028-cross-chain-validation/overview_and_basic_concepts.md

```diff
@@ -0,0 +1,119 @@
+<!-- omit in toc -->
+# CCV: Overview and Basic Concepts
+[&uparrow; Back to main document](./README.md)
+
+<!-- omit in toc -->
+## Outline
+- [Security Model](#security-model)
+- [Motivation](#motivation)
+- [Definitions and Overview](#definitions-and-overview)
+  - [Channel Initialization](#channel-initialization)
+  - [Validator Set Update](#validator-set-update)
+
+
+
+# Security Model
+[&uparrow; Back to Outline](#outline)
+
+We consider chains that reach consensus through a proof of stake mechanism based on the model of [weak subjectivity](https://blog.ethereum.org/2014/11/25/proof-stake-learned-love-weak-subjectivity/). 
+The next block in a blockchain is *validated* and *voted* upon by a set of pre-determined *full nodes*, also known as *validators*. 
```

Comment:
Careful about terminology. _full node_ and _validator_ is not exactly the same (not all full nodes are validators, at least in Tendermint terminology)

Reply:
The "pre-determined *full nodes*" that "*validate* and *vote* the next block"  are  "also known as *validators*". But I get how the phrasing may be confusing.

Reply:
```suggestion
The next block in a blockchain is *validated* and *voted* upon by a set of pre-determined *full nodes*; these pre-determined full nodes also known as *validators*. 
```